frontend/src/components/editor-page/editor-pane/hooks/yjs/use-receive-realtime-users.ts

```diff
@@ -21,7 +21,12 @@ export const useReceiveRealtimeUsers = (messageTransporter: MessageTransporter):
   useEffect(() => {
     const listener = messageTransporter.on(
       MessageType.REALTIME_USER_STATE_SET,
-      (payload) => setRealtimeUsers(payload.payload),
+      (payload) =>
+        setRealtimeUsers(
+          payload.payload.users,
+          payload.payload.ownUser.styleIndex,
+          payload.payload.ownUser.displayName
```

Comment:
```suggestion
      (message) =>
        setRealtimeUsers(
          message.payload.users,
          message.payload.ownUser.styleIndex,
          message.payload.ownUser.displayName
```

`payload.payload` sounds weird (and yes I know that `message` is not the complete message